lib/perl/Genome/Sys.pm

```diff
@@ -1522,10 +1528,15 @@ sub shellcmd {
                 {   # POE sets a handler to ignore SIG{PIPE}, that makes the
                     # pipefail option useless.
                     local $SIG{PIPE} = 'DEFAULT';
-                    $system_retval = system('bash', '-c', "$shellopts_part $cmd");
+                    my @cmdline = ('bash', '-c', "$shellopts_part $cmd");
+                    exec(@cmdline)
+                        or do {
```

Comment:
I suppose the `or do` isn't even necessary.  The next two lines after `exec` could just be the `print` and `exit` since they will never run if the `exec` works.  I'm not sure if this idea is good or bad...